member/index.md

```diff
@@ -94,7 +94,8 @@ We'd like you to bear in mind that none of our **moderators** work for nor are a
 ### Misc Commands
 | Command                 | Function                                                 |
 | ----------------------- | -------------------------------------------------------- |
-| !rules                  | Posts a link to the server rules                         |
+| !rules                  | Posts a snippet of all rules                             |
```

Comment:
This actually wouldn't work rigth now, it requires `all` argument, but I am going to change it soon™, so that it works.

Reply:
You can keep this as is.

Reply:
Oh! I haven't noticed that this changed but it made sense as it was a very big snippet. I've kept this for now though